ToolTip.ios.js

```diff
@@ -24,15 +24,19 @@ var propTypes = {
 var ViewClass = React.createClass({
   getDefaultProps: function() {
     return {
-      arrowDirection: 'down'
+      arrowDirection: 'down',
+      onHide: () => true,
+      onShow: () => true
     };
   },
 
   showMenu: function() {
     ToolTipMenu.show(findNodeHandle(this.refs.toolTipText), this.getOptionTexts(), this.props.arrowDirection);
+    this.props.onShow();
```

Comment:
`showMenu` here is only used when programmatically displaying the tooltip. about ur usecase:

y can't you do something like this:
```
handleLongPress() {
  showMenu();
  this.setState({ style: { ...} })
}
```

that's just an example. I'm trying to understand the use case better.